app/controllers/checklist_tests_controller.rb

```diff
@@ -6,9 +6,7 @@ class ChecklistTestsController < ProductTestsController
   before_action :set_measures, only: [:show]
 
   def create
-    # bundle ATTRIBUTE IS NOT CHOSEN CORRECTLY. MUST FIX LATER ~ JAEBIRD
```

Comment:
Was thinking of another class that isn't used (filtering_test_controller). But my other statement is still relevant.